operations/node_management/Rebuild_NCNs/Prepare_Storage_Nodes.md

```diff
@@ -47,30 +50,11 @@ When rebuilding a node, make sure that `/srv/cray/scripts/common/storage-ceph-cl
 
 Upload Ceph container images into nexus.
 
-1. (`ncn-m001#`) Copy `/usr/share/doc/csm/scripts/upload_ceph_images_to_nexus.sh` to **the node being rebuilt** and execute it.
-
-    ```bash
-    scp /usr/share/doc/csm/scripts/upload_ceph_images_to_nexus.sh ${NODE}:/srv/cray/scripts/common/upload_ceph_images_to_nexus.sh
-    ssh $NODE /srv/cray/scripts/common/upload_ceph_images_to_nexus.sh
-    ```
-
-1. (`ncn-s00[1/2/3]#`) After running the script, run the following command to check for errors or completion of the `ceph orch upgrade` command run in the script.
+1. Logged into one of (`ncn-s00[1/2/3]`), execute the `upload_ceph_images_to_nexus.sh`.
 
-    ```bash
-    ceph orch upgrade status
-    ```
-
-    Expected output:
-
-    ```bash
-    {
-    "target_image": null,
-    "in_progress": false,
-    "services_complete": [],
-    "progress": null,
-    "message": ""
-    }
-    ```
+   ```bash
+   /srv/cray/scripts/common/upload_ceph_images_to_nexus.sh
```

Comment:
This file should be copied over from m001 since storage nodes won't have the most recent version of this file in their node image

Reply:
the docs rpm is installed on nodes as part of the upgrade prerequisites.

Reply:
I see it being uploaded to nexus here: https://github.com/Cray-HPE/docs-csm/blob/release/1.4/upgrade/scripts/upgrade/prerequisites.sh#L370-L386

I don't see where we install it on the other NCNs, maybe I'm missing it?

Reply:
AFAIK it isn't installed on the other nodes, but things may have changed since the last time I looked at the update code.

Reply:
I did some 1.4 upgrades last week and it definitely was not installed on any other node outside of the manual steps of installing it on m001 and m002.

Reply:
This has been fixed by commit [c9f78f8](https://github.com/Cray-HPE/docs-csm/pull/3592/commits/c9f78f8f3eb374b6df8efd6696e93b1c88c4b388)